application/views/templates/footer.php

```diff
@@ -8,8 +8,9 @@
 </script>
 
 
-<script src="<?php echo base_url();?>assets/js/jquery.slim.min.js"></script>
+<script src="<?php echo base_url();?>assets/js/jquery-3.6.3.js"></script>
 <script src="<?php echo base_url();?>assets/js/bootstrap.bundle.min.js"></script>
+<script src="<?php echo base_url();?>assets/js/mycustom.js"></script>
```

Comment:
Are you still using this for post request? Try using the todo-module.js instead.

Reply:
i forgot to delete this. on my next push this will be deleted
im now using the todo-module